meta-mender-beaglebone/templates/local.conf.append

```diff
@@ -0,0 +1,7 @@
+
+# Appended fragment from meta-mender-community/meta-mender-beaglebone/templates
+
+MACHINE ?= "beaglebone-yocto"
+
+MENDER_BOOT_PART_SIZE_MB = "40"
```

Comment:
Did you have issues with the default values for above? Would not expect this to be required on BBB.